readme.md

```diff
@@ -126,6 +126,12 @@ If you are on Arch and would like to help with a PR that I can get merged into m
 ```bash
 sudo pacman -S xdotool dmenu xorg-server xorg-xwininfo xorg-xrandr protobuf base-devel zeromq libx11 libxcomposite libxtst libxext libxfixes spdlog fmt glfw-x11 mesa assimp sqlite
 ```
+#### Gentoo
+```bash
+ sudo emerge net-libs/zeromq x11-libs/libX11 x11-libs/libXcomposite x11-libs/libXtst x11-libs/libXext x11-libs/libXfixes dev-libs/protobuf dev-libs/spdlog dev-libs/libfmt glfw x11-libs/libGLw  dev-db/sqlite x11-misc/xdotool  protobuf dev-libs/pthreadpool media-libs/libass media-lib/assimp
+```
+> [!NOTE]
+> you may have some isses with use flags and or masked packages. you will have to figer that out on your own system.
```

Comment:
I'll change this to say 
" you may have some issues with use-flags and or masked packages, you will have to figure it out on your system as your use-flag configuration may differ from others systems."

Reply:
replace "figer" with "figure" 

"s/figer/figure" is how I normally comment about a substitution

Reply:
Ok I'll fix it